docs/reference/commandline/swarm_init.md

```diff
@@ -26,6 +26,7 @@ Options:
       --availability string             Availability of the node ("active"|"pause"|"drain") (default "active")
       --cert-expiry duration            Validity period for node certificates (ns|us|ms|s|m|h) (default 2160h0m0s)
       --data-path-addr string           Address or interface to use for data path traffic (format: <ip|interface>)
+      --default-addr-pool string	List of default subnet addresses followed by subnet size (format: <subnet,subnet,..:subnet-size)
```

Comment:
Should this option also be on `swarm update`? (i.e., is it possible to change default address pools without having to tear down the cluster?)

Reply:
that is P1.  once I finish all P0 tasks, I will start working on P1.

Reply:
nit:  no closing '>'  Actually, maybe the format should be `cidr[,cidr]*:masklen`. 

In general, a problem I'm having is that we refer to that last parameter as the "subnet size".  But that's actually misleading.  It's the length of the mask that accordingly defines the subnet size.  This applies to parts of the code as well.

Maybe clearer terminology would be
  * "address block" or "address range" -- contiguous address space that will be carved up into allocatable subnets
  * "subnet mask length" or just "mask length" -- the mask length for all subnets in the global address pool

These are a bit more verbose.  I dunno.  Maybe the existing terminology is more clear to others.  Would welcome comment.

Reply:
I will remove "<"  and change the format . WRT subnet size, the same name has been used in existing code and changing it will again mislead having two different name which points to same attribute.